docs/ecosystem-eslint-plugin-testing-library.md

```diff
@@ -0,0 +1,23 @@
+---
+id: ecosystem-eslint-plugin-testing-library
+title: eslint-plugin-testing-library
+---
+
+[`eslint-plugin-testing-library`][gh] is an ESLint plugin for `Testing Library`
```

Comment:
`Testing Library` does not need to be in backticks I think.

Reply:
done too